src/status.js

```diff
@@ -56,25 +56,35 @@ export default class Status extends Component {
       content = tResults(length, contentSelectedOption)
     }
 
-    return <div
-      aria-atomic='true'
-      aria-live='polite'
-      role='status'
-      style={{
-        border: '0',
-        clip: 'rect(0 0 0 0)',
-        height: '1px',
-        marginBottom: '-1px',
-        marginRight: '-1px',
-        overflow: 'hidden',
-        padding: '0',
-        position: 'absolute',
-        whiteSpace: 'nowrap',
-        width: '1px'
-      }}
-    >
-      {content}
-      <span>{bump ? ',' : ',,'}</span>
-    </div>
+    return (
+      <div
+        style={{
+          border: '0',
+          clip: 'rect(0 0 0 0)',
+          height: '1px',
+          marginBottom: '-1px',
+          marginRight: '-1px',
+          overflow: 'hidden',
+          padding: '0',
+          position: 'absolute',
+          whiteSpace: 'nowrap',
+          width: '1px'
+        }}>
+        <div
+          id='ariaLiveA'
+          role='status'
+          aria-atomic='true'
+          aria-live='polite'>
+          <span>{bump ? content : ''}</span>
+        </div>
+        <div
+          id='ariaLiveB'
+          role='status'
+          aria-atomic='true'
+          aria-live='polite'>
+          <spam>{!bump ? content : ''}</spam>
```

Comment:
Is it possible to fix this typo?

Reply:
Dammit! .. ha ha, how the hell did I miss that.

Reply:
Ok done.

Reply:
Hey Nick, cheers for checking over the commits, appreciated.

Happy to help with issue categorisation where I can.  May have to step back from the detail for a while, but feel free to ping me with plan of attack. :thumbsup: